filesystem/src/main/java/org/frankframework/filesystem/FileSystemActor.java

```diff
@@ -291,8 +290,9 @@ private F getFile(@Nonnull Message input, ParameterValueList pvl) throws FileSys
 	private F getFileAndCreateFolder(@Nonnull Message input, ParameterValueList pvl) throws FileSystemException {
 		String filenameWithFolder = determineFilename(input, pvl);
 		String folder = FilenameUtils.getFullPathNoEndSeparator(filenameWithFolder);
-		if(StringUtils.isNotBlank(folder) && isCreateFolder() && !fileSystem.folderExists(folder)) {
-			fileSystem.createFolder(folder);
+		if(StringUtils.isNotBlank(folder) && !fileSystem.folderExists(folder)) {
```

Comment:
```suggestion
		if (StringUtils.isNotBlank(folder) && !fileSystem.folderExists(folder)) {
			if (isCreateFolder()) {
				fileSystem.createFolder(folder);
			} else {
				throw new FolderNotFoundException("folder ["+folder+"] does not exist");
			}
		}
```

Reply:
You want more braces.. I wanted to keep it more compact. 😉 

I cannot commit the suggestion this way I think, since that would mess up the rest of the change in the file.